hashicorp/dev-portal
# feat: product page data scaffold

## "Ready for Review" checklist




---

## Relevant links



- [Preview link](https://dev-portal-git-ksproduct-page-data-scaffold-hashicorp.vercel.app/waypoint/tutorials) 🔎
- [Asana task](https://app.asana.com/0/1202001387788447/1202117507047614) 🎟️

## What

This PR adds data fetching for product pages and sets up the basic layout scaffolding. 

## How

I mostly copied @zchsh's awesome code from the Learn codebase 😄 I assume we'll continue with the blocks based approach that was set-up a few months ago, cause it rules. This PR gets all that data in place and ready to be styled. 

## Testing

- Visit [vault product tutorials hub](https://dev-portal-git-ksproduct-page-data-scaffold-hashicorp.vercel.app/vault/tutorials)
- Visit [waypoint product tutorials hub](https://dev-portal-git-ksproduct-page-data-scaffold-hashicorp.vercel.app/waypoint/tutorials)

Make sure the sidebars match learn.hashicorp.com and that the block headers also match the learn counterparts for [waypoint](https://learn.hashicorp.com/waypoint) and [vault](https://learn.hashicorp.com/vault). 

## Anything else?

Nope! I didn't copy over the blocks because I figure we'll want to do that at the time of styling.
